src/zennit/layer.py

```diff
@@ -34,3 +34,52 @@ def __init__(self, dim=-1):
     def forward(self, input):
         '''Computes the sum along a dimension.'''
         return torch.sum(input, dim=self.dim)
+
+
+class Distance(torch.nn.Module):
+    '''Compute distance between inputs and centroids.'''
+    def __init__(self, centroids, power=2):
+        super().__init__()
+        self.centroids = torch.nn.Parameter(centroids)
```

Comment:
Perhaps, we could add a sentence the docstring describing the expected dimensions of  `centroids`.